wagtailio/static/sass/components/_nav-cta.scss

```diff
@@ -0,0 +1,96 @@
+.nav-cta {
+    $root: &;
+    display: block;
+    position: relative;
+    color: $color--white;
+    text-decoration: none;
+    padding: 20px 20px 50px 20px;
+
+    @include media-query(medium) {
+        padding: 20px 50px 50px 50px;
+    }
+
+    @media only screen and (min-width: $header-desktop) {
+        padding: 60px 60px 100px 60px;
+    }
+
+    &:hover,
+    &:focus {
+
+        #{$root}__cta-heading {
+            color: $color--teal-100;
+        }
+
+    }
+
+    &__icon {
+        @include z-index(base);
+        position: absolute;
+        top: 30px;
+        right: 20px;
+        width: 190px;
+        height: auto;
+        color: rgba($color--teal-200, 0.75);
+
+        @media only screen and (min-width: $header-desktop) {
+            top: 60px;
+            right: 60px;
+        }
+    }
+
+    &__content {
+        @include z-index(overlap);
+        position: relative;
+        padding-top: 40px;
+        text-align: start;
+
+        @media only screen and (min-width: $header-desktop) {
+            padding-top: 100px;
+        }
+    }
+
+    &__heading {
+        font-size: 45px;
+        line-height: 1.1;
+        margin-bottom: 10px;
+        padding-right: 60px;
+        transition: color $transition;
```

Comment:
question: is `color` set somewhere else? Can't see it in the list of properties listed here

Reply:
it's inherited by the main .nav-cta and a hover applied on the same selector